schedule.JSON

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
Couple of things with this file:

1. it's common to use lowercase for the file extension, so see if you can rename the file to `schedule.json`.
2. since `games` is the only key, it probably means that you can just simplify the file to not use an object, and instead just use the value of the `games` property, i.e.:

```
[
  {
    "date": "...",
    "...": "..."
  },
  {
    "date": "...",
    "...": "..."
  }
]
```

It's perfectly valid for JSON to be just an array without being wrapped in some object. Notice how doing this will simplify the code that uses it because you'll no longer need that weird `jschedule` variable.

Reply:
I see what you are saying, however my intention was to later add the standings into the JSON file as well so I figured it would be easier to just do add the key now rather than change it later.

Reply:
The part I don't think I understand it, wouldn't I need to parse the JSON data anyway? How would that remove the necessity for the jschedule variable?

Reply:
I think your most recent commits were a regression. I would definitely _not_ join the schedule and the standings into the same JSON file. Why? Couple of reasons:

1. If you join them, the user must necessarily download both, even if they might only visit, say, the schedule.
2. They're two logically separate things, so why not keep them separate? I don't see a real good reason to keep them together.

As for eliminating the `jschedule` variable, rather than this:

```json
{
  "games": [
    // ...
  ]
}
```
```js
const jschedule = JSON.parse(schedule.responseText)
const htmlString = jschedule.games.map(/*...*/)
```

you can just do this:
```json
[
  // ...
]
```
```js
const games = JSON.parse(schedule.responseText)
const htmlString = games.map(/*...*/)
```

Reply:
I guess its sort of like lines 20 and 21 in my new standings.js file, except I would no longer need that middle man of jstandings?